mobile/library/java/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java

```diff
@@ -208,17 +200,21 @@ String resolveTemplate(final String configTemplate, final String platformFilterT
     }
 
     if (enableHttp3) {
+      final String altProtocolCacheFilterInsert = JniLibrary.altProtocolCacheFilterInsert();
```

Comment:
Out of curiosity, did you consider eliminating the local variable and just doing: `customFiltersBuilder.append(JniLibrary.altProtocolCacheFilterInsert());`? I think that might be a bit shorter and simpler.

Reply:
true but again the end goal is to remove all of this and I'm trying to minimize diffs in between.  Given that, OK to leave as is?